activestorage/db/migrate/20180723000244_add_foreign_key_constraint_to_active_storage_attachments_for_blob_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddForeignKeyConstraintToActiveStorageAttachmentsForBlobId < ActiveRecord::Migration[6.0]
+  def change
+    add_foreign_key :active_storage_attachments, :active_storage_blobs, column: :blob_id
```

Comment:
Hm. For some reason, this doesn't add the constraint.
https://travis-ci.org/rails/rails/jobs/406956010#L1499
I'll look at this later

Reply:
It is probably a limitation for SQLite3, SQLite3 doesn't support ALTER CONSTRAINT.

Reply:
I see. Not sure there is something we should do with it.

How about adding execution of ActiveStorage's tests with MySQL and PostgreSQL to the travis matrix (and probaly skip failed test for SQLite3)?